cns/restserver/util.go

```diff
@@ -590,7 +590,9 @@ func logNCSnapshot(createNetworkContainerRequest cns.CreateNetworkContainerReque
 	aiEvent.Properties[logger.NetworkContainerTypeStr] = createNetworkContainerRequest.NetworkContainerType
 	aiEvent.Properties[logger.OrchestratorContextStr] = fmt.Sprintf("%s", createNetworkContainerRequest.OrchestratorContext)
 
-	// TODO - Add for SecondaryIPs (Task: https://msazure.visualstudio.com/One/_workitems/edit/7711831)
+	if service.state.OrchestratorType == cns.KubernetesCRD {
```

Comment:
Does reading `state` need a read lock from `service`? Would it be simpler to just log if len of secondary ip configs > 0?